templates/paloalto_panos_test_security-policy-match.textfsm

```diff
@@ -0,0 +1,36 @@
+Value Filldown NAME (.*?)
+Value INDEX (\S+)
+Value Required FROM (\S+)
+Value SOURCE (\S+)
+Value SOURCE_REGION (\S+)
+Value TO (\S+)
+Value DESTINATION ([\S+\s+]+)
+Value DESTINATION_REGION (\S+)
+Value USER (\S+)
+Value SOURCE_DEVICE (\S+)
+Value DESTINATION_DEVICE (\S+)
+Value CATEGORY (\S+)
+Value APPLICATION_SERVICE (\[.*\])
```

Comment:
I don't think APPLICATION_SERVICE is capturing as you expect from the `.raw` and `yml` so may need to revisit this Regex.

Reply:
Spot on on this one, I've fixed it.  I was using the regex from the running config that didn't work in this case.  Also the same one your previous IMPLICIT comment, that doesn't exist in the test command output.  Thanks for checking my work.